docs/resources/as_policy.md

```diff
@@ -97,55 +101,70 @@ resource "huaweicloud_as_policy" "my_aspolicy_2" {
 
 The following arguments are supported:
 
-* `region` - (Optional, String, ForceNew) The region in which to create the AS policy. If omitted, the `region` argument
-  of the provider is used. Changing this creates a new AS policy.
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the AS policy. If omitted, the
+  provider-level region will be used. Changing this creates a new AS policy.
 
-* `scaling_policy_name` - (Required, String) The name of the AS policy. The name can contain letters, digits,
+* `scaling_policy_name` - (Required, String) Specifies the name of the AS policy. The name contains only letters, digits,
   underscores(_), and hyphens(-), and cannot exceed 64 characters.
 
-* `scaling_policy_type` - (Required, String) The AS policy type. The values can be `ALARM`, `SCHEDULED`,
-  and `RECURRENCE`.
+* `scaling_group_id` - (Required, String, ForceNew) Specifies the AS group ID. Changing this creates a new AS policy.
 
-* `scaling_group_id` - (Required, String, ForceNew) The AS group ID. Changing this creates a new AS policy.
+* `scaling_policy_type` - (Required, String) Specifies the AS policy type. The value can be `ALARM`, `SCHEDULED`,
+  and `RECURRENCE`.
+  + **ALARM**: indicates that the scaling action is triggered by an alarm.
```

Comment:
how about "indicates that the scaling action will be triggered by an alarm"?

Reply:
this description  is derived from [support](https://support.huaweicloud.com/intl/en-us/api-as/as_06_0401.html)